page-objects/pages/coursePages/CRSSAddEditTopicPage.js

```diff
@@ -72,11 +72,16 @@ async function addEtherpad(name, description) {
 	await waitHelpers.waitAndSetValue(etherpadDescriptionField, description);
 }
 
-async function isTopicOnTopicList(name) {
-	const listOfTopicTitles = await topicTitleList();
-	const msg = 'Topic with name [' + name + '] is not visible on the list \n';
-	const resultMsg = ', List of task titles: ' + listOfTopicTitles;
-	expect(listOfTopicTitles, msg + resultMsg).to.include(name);
+async function isTopicOnTopicList(topicname, expectedValue) {
+	const allTopics = await topicTitleList();
+    const isTopicOnList = allTopics.some((element) => element.includes(topicname));
+    const fillString = !expectedValue ? 'not' : '';
+	const msg = `Topic with name is ${fillString} visible on the list: \n`;
+	const resultMsg = 'Expected: ' + topicname + ', Actual: ' + allTopics;
+
+	expectedValue
```

Comment:
I need an explanation for that. Sorry:
`expectedValue
		? await expect(isTopicOnList, msg + resultMsg).to.equal(true)
		: await expect(isTopicOnList, msg + resultMsg).to.equal(false);`

Reply:
This way we can handle the check of a new creation and edit with true and the deletion with false.
I took this from the courses page, where we use it in a similar way.